helix-term/src/commands.rs

```diff
@@ -3044,6 +3045,9 @@ pub mod cmd {
         });
 }
 
+static FUZZY_MATCHER: Lazy<fuzzy_matcher::skim::SkimMatcherV2> =
```

Comment:
minor: Since this is only used inside `fn command_mode` I'd move the definition inside the `fn` body, that way it's scoped to the command.